state/protocol/badger/mutator_test.go

```diff
@@ -671,6 +671,148 @@ func TestExtendHighestSeal(t *testing.T) {
 	})
 }
 
+func TestExtendReceiptsDuplicate(t *testing.T) {
```

Comment:
could we please also include a test that verifies that receipts for the latest sealed block (in the respective fork) are not accepted.